astropy/coordinates/tests/test_earth.py

```diff
@@ -410,3 +437,236 @@ def test_info():
     EarthLocation._get_site_registry(force_builtin=True)
     greenwich = EarthLocation.of_site('greenwich')
     assert str(greenwich.info).startswith('name = Royal Observatory Greenwich')
+
+
+class TestWGS84GeodeticRepresentationRepresentation:
```

Comment:
I would shrink this class substantially - many of the tests are done on `Spherical` to just have them done once, no need to do them again. Also, do re-align for the longer name...

Reply:
Thinking more, since all new tests are (very nicely) just focussed on the representations, how about a new `test_geodetic_representations`?

Reply:
I moved these tests to `test_geodetic_representations.py`, but I'm not sure which ones to remove.